examples/build-package/main.tf

```diff
@@ -278,8 +302,31 @@ module "lambda_layer" {
 
   build_in_docker = true
   runtime         = "python3.8"
-  docker_image    = "public.ecr.aws/sam/build-python3.8"
+  docker_image    = "build-python3.8"
```

Comment:
If `public.ecr.aws/sam/build-python3.8` already exists locally, the `docker_file` is not used.